test/scripts/openshift-ci/infra/deploy.lws.sh

```
@@ -19,34 +19,34 @@ source "$SCRIPT_DIR/../common.sh"
 
 echo "⏳ Installing openshift-lws-operator"
 
-oc create ns openshift-lws-operator || true
+oc create ns ${LWS_NAMESPACE} || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)